### ISSUE TITLE
create flag that separates decorative from informative or functional images

### ISSUE THREAD

**Issue by dieMelanie** [marketing-factory/ai-filemetadata#3]
Cited from W3C:
* Decorative images don’t add information to the content of a page. In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers.
* Informative images convey a simple concept or information that can be expressed in a short phrase or sentence.
* Functional images are used to initiate actions rather than to convey information
The text alternative for the image should convey the action that will be initiated (the purpose of the image), rather than a description of the image.
* Complex images contain substantial information – more than can be conveyed in a short phrase or sentence. These are typically: graphs and charts, including flow charts and organizational charts;
diagrams and illustrations where the page text relies on the user being able to understand the image;
maps showing locations or other information such as weather systems.
